R/pin-read-write.R

```diff
@@ -30,9 +33,31 @@
 #'
 #' # can use `version` argument to read a specific version:
 #' pin_versions(model_board, "cars_linear")
+#' @examplesIf interactive() || identical(Sys.getenv("IN_PKGDOWN"), "true")
+#' # can store an renv lockfile as part of the pin:
+#' vetiver_pin_write(model_board, v, check_renv = TRUE)
 #'
 #' @export
-vetiver_pin_write <- function(board, vetiver_model, ...) {
+vetiver_pin_write <- function(board, vetiver_model, ..., check_renv = FALSE) {
+
+    renv_lock <- NULL
+
+    if (check_renv) {
+        pkgs <- c(vetiver_model$metadata$required_pkgs, "vetiver")
```

Comment:
This does _not_ include all the packages needed for the API like plumber, pins, etc but I think that is the right decision. That makes these the packages needed for the _model only_ (not necessarily the deployment).